@here/harp-datasource-protocol/test/StyleSetEvaluatorTest.ts

```diff
@@ -989,8 +990,8 @@ describe("StyleSetEvaluator", function () {
                 const technique = techniques[0] as PoiTechnique;
 
                 const context: AttrEvaluationContext = {
-                    env,
-                    cachedExprResults: styleSetEvaluator.expressionEvaluatorCache
+                    env
+                    // cachedExprResults: styleSetEvaluator.expressionEvaluatorCache
```

Comment:
I'd propose to remove this commented out line. You removed "styleSetEvaluator.expressionEvaluatorCache" so here you refer to the property that doesn't exist in the code anymore, it will be confusing for future us.

Reply:
done